### PR TITLE
wip: still on objects.js

### PR DESCRIPTION
**Pull request by snellback**
Completed arrays.js

### CODE REVIEW

File: assignments/objects.js [hunk 1]
```diff
@@ -18,25 +18,62 @@ const example = {
 }
 
 // Write your intern objects here:
+const mitzi = {
```
Comment (Mykolb) [inline assignments/objects.js:21]: Objects look good!

File: assignments/arrays.js [hunk 2]
```diff
@@ -63,32 +63,55 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
...
+    return output.sort().toString();
+}
+
+console.log(carModels(inventory, "car_model"))
 
 // ==== Challenge 4 ====
 // The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
-let carYears = [];
-console.log();
+let carYears = []
```
Comment (Mykolb) [inline assignments/arrays.js:94]: Good job! Everything ran without errors.

**Comment by Mykolb** [comment]
You just need the array function conversions and you are done :-)

**Comment by snellback** [comment]
Finished

File: assignments/function-conversion.js [hunk 3]
```diff
@@ -13,16 +13,25 @@ myFunction();
 // };
 // anotherFunction("Example");
 
+let anotherFunction = (param) => param;
```
Comment (Mykolb) [inline assignments/function-conversion.js:16]: Great job! 

When you use an arrow function and have one param, you don't need the ().

**Comment by Mykolb** [comment]
MVP met, merging PR

File: assignments/function-conversion.js (hunk 3 above)
Comment (snellback) [reply assignments/function-conversion.js:16]: Ok

**Pull request merged**